src/sentry/workflow_engine/handlers/condition/group_event.py

```diff
@@ -24,6 +24,14 @@ def get_nested_value(data: Any, path: str, default: Any = None) -> Any | None:
 @condition_handler_registry.register(Condition.GROUP_EVENT_ATTR_COMPARISON)
 class GroupEventConditionHandler(DataConditionHandler[GroupEvent]):
     @staticmethod
-    def evaluate_value(data: GroupEvent, comparison: Any, data_filter: str) -> bool:
-        event_value = get_nested_value(data, data_filter)
+    def evaluate_value(
+        value: GroupEvent, comparison: Any, condition: str, **kwargs: Any
+    ) -> DetectorPriorityLevel | int | float | bool | None:
+        # conditions stores the path to the attribute in the event
+        event_value = get_nested_value(value, condition)
+        if event_value is None:
+            event_value = get_nested_value(
+                kwargs, condition
+            )  # TODO: remove when GroupEvent contains EventState
```

Comment:
do we need to split here? could we just invoke the condition with the kwargs as the value? (if it's a different input type, we can add another method here with a different type and reuse the `get_nested_value` method in that handler)

Reply:
i put this here in the case that we actually do decide to move these out of a separate kwarg and into the `GroupEvent` itself